apps/application-system/api/src/app/modules/application/template-api.service.ts

```diff
@@ -1,19 +1,21 @@
 import { Inject, Injectable } from '@nestjs/common'
-import { ApplicationService } from './application.service'
 import { ApplicationWithAttachments } from '@island.is/application/core'
-import { BaseTemplateApiApplicationService } from '@island.is/application/template-api-modules'
+import {
+  BaseTemplateApiApplicationService,
+  TemplateAPIConfig,
+} from '@island.is/application/template-api-modules'
+import { ApplicationService } from '@island.is/application/nest/application'
+import { AwsService } from '@island.is/nest/aws'
+import { ConfigService } from '@nestjs/config'
 import { uuid } from 'uuidv4'
-import { AwsService } from './files'
-import type { ApplicationConfig } from './application.configuration'
-import { APPLICATION_CONFIG } from './application.configuration'
 
 @Injectable()
 export class TemplateApiApplicationService extends BaseTemplateApiApplicationService {
   constructor(
     private readonly applicationService: ApplicationService,
     private readonly awsService: AwsService,
-    @Inject(APPLICATION_CONFIG)
-    private readonly config: ApplicationConfig,
+    @Inject(ConfigService)
+    private readonly configService: ConfigService<TemplateAPIConfig>,
```

Comment:
Wondering if there is an opportunity to refactor the config setup of the modules to use our ConfigModule wrapper of the Nest Config?
https://docs.devland.is/libs/nest/config

Reply:
yeah absolutely. Onit!

Reply:
On second thought want this in for the next release. Going to add this to our backlog and another PR later on. If your ok with that.

Reply:
Yep that is fine.